packages/fast-components-react-base/src/viewport-positioner/viewport-positioner.tsx

```diff
@@ -327,6 +327,9 @@ class ViewportPositioner extends Foundation<
      *  gets the CSS classes to be programmatically applied to the component
      */
     private getPositioningStyles = (): React.CSSProperties => {
+        // Check if there is already a style object being passed as props
+        const styleProps: React.CSSProperties = get(this.props, "style");
```

Comment:
It seems to me that since this change is happening in `@microsoft/fast-components-react-base` it will add to the changelog a minor bump and the item "add flyout as a new component". This doesn't seem desirable since this should be a change to msft packages only. Can we split the changes here into another PR?